core/pom.xml

```diff
@@ -284,10 +258,20 @@
       <artifactId>jetty-servlet</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.eclipse.jetty</groupId>
+      <artifactId>jetty-util</artifactId>
+      <scope>test</scope>
+    </dependency>
     <dependency>
       <groupId>com.google.errorprone</groupId>
       <artifactId>error_prone_annotations</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.dataformat</groupId>
+      <artifactId>jackson-dataformat-smile</artifactId>
+      <scope>test</scope> <!-- compile -->
```

Comment:
What does this comment mean?

Reply:
Maven does not have different scopes to distinguish between test compile and test runtime (unlike gradle). I can improve the comment, but since it's not used consistently perhaps it's better to remove.

Reply:
Ah got it. Yeah I would be fine with either a more descriptive comment (like "only needed for test compilation, not runtime") or just removing it.